openfasoc/generators/common/get_ngspice_version.py

```diff
@@ -2,6 +2,7 @@
 import re
 
 def check_ngspice_version() -> int:
+    
```

Comment:
Seems unnecessary.

Reply:
Added this just so we can eliminate the ngspice version as a variable if something goes wrong in the simulation CI checks

Reply:
I meant the extra newline, sorry.